test/unittests/libp2p/net.cpp

```diff
@@ -546,7 +552,8 @@ BOOST_AUTO_TEST_CASE(noteActiveNodeReplacesNodeInFullBucketWhenEndpointChanged)
     // but added as most recently seen with new endpoint
     auto mostRecentNodeEntry = nodeTable->bucketLastNode(bucketIndex);
     BOOST_CHECK_EQUAL(mostRecentNodeEntry->id, leastRecentlySeenNodeId);
-    BOOST_CHECK_EQUAL(mostRecentNodeEntry->endpoint, newEndpoint);
+    BOOST_CHECK_EQUAL(mostRecentNodeEntry->endpoint.address(), newEndpoint.address());
```

Comment:
I changed the meaning of this test for now, but I think this whole case of changed endpoint should be revisited in https://github.com/ethereum/aleth/issues/5455, and this test probably will be removed.